ee/clickhouse/queries/funnels/test/test_funnel_persons.py

```diff
@@ -305,10 +317,35 @@ def test_first_step_breakdowns(self):
         )._exec_query()
         self.assertCountEqual([val[0] for val in results], [person2.uuid])
 
+    def test_first_step_breakdowns_with_multi_property_breakdown(self):
+        person1, person2 = self._create_browser_breakdown_events()
```

Comment:
non-blocking

I find these types of fixture functions quite opaque, is inlining the generation or making it more explicit at a glance the important details an option?

Reply:
🤔 possibly... I can't immediately see the better option. That method is shared between three tests. Probably the right thing is to analyse whether the setup data for each of the tests is different or not...